drivers/hud_spm_report/app/models/hud_spm_report/generators/fy2020/base.rb

```diff
@@ -75,8 +75,8 @@ def she_household_column
       project_id: p_t[:id],
       project_name: :project_name,
       household_id: she_household_column,
-      housing_status_at_entry: :housing_status_at_entry,
-      housing_status_at_exit: :housing_status_at_exit,
+      housing_status_at_entry: :housing_status_at_entry, # NOTE: use Enrollment.LivingSituation in the future
+      housing_status_at_exit: :housing_status_at_exit, # NOTE: use Exit.HousingAssessment in the future
```

Comment:
?

Reply:
This is to remind us when we write the 2023 SPM (and copy the code from here) that we shouldn't be using that value any more.

Reply:
Hmm, the previous SPM specs say:
System-specific tag for identifying projects serving Category 3 homeless
Dependent on specific HMIS product.
No current HMIS Data Standards indicator.

So maybe we came up with that?